README.md

```diff
@@ -1,6 +1,10 @@
-mpsc\_requests
+crossbeam\_requests
 =============
 
-[![test](https://docs.rs/mpsc_requests/badge.svg)](https://docs.rs/mpsc_requests/0.1.0/mpsc_requests/)
+[![test](https://docs.rs/crossbeam_requests/badge.svg)](https://docs.rs/crossbeam_requests/latest/crossbeam_requests/)
 
 For more info look at the docs linked above
+
+TODO:
+- Rename back to mpsc_requests? crossbeam-requests is an undescriptive name and mpsc_request is inferior.
```

Comment:
Why rename? crossbeam-requests seems descriptive enough for me?

Reply:
Because crossbeam_requests is just an implementation detail and tells nothing about what this library actually does. mpsc_requests tells exactly what it does, multiple producers can make a request to a single consumer. crossbeam consists of more than just mpsc/mpmc channels.